R/read_spe.R

```diff
@@ -0,0 +1,425 @@
+# Reading of SPE files, produced by Princeton Instruments spectrometers
+# File format version 2.5 (Sept. 2002)
+
+# C. Beleites
+# R. Kiselev
```

Comment:
It is clear, that Roman Kiselev (@ximeg) must be included among the authors of this package.

Reply:
Yes, I added him to the description.md file.